src/randoop/main/ClassTypeLoader.java

```diff
@@ -0,0 +1,134 @@
+package randoop.main;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import randoop.types.TypeNames;
+import randoop.util.Files;
+
+import plume.EntryReader;
+
+/**
+ * ClassTypeLoader provides methods that allow a list of class names to be loaded
```

Comment:
You mean loaded into the JVM?  I would clarify that.  (When I first read it, I thought it might be loading from a file.)